2021-11-17-qubes-4-1-rc2.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Qubes OS 4.1-rc2 has been released!"
+categories: releases
+download_url: /downloads/
+author: Andrew David Wong
+---
+
+We're pleased to announce the second release candidate for Qubes 4.1!
+
+Qubes 4.1-rc2 contains fixes for bugs that were discovered in the first
+release candidate (4.1-rc1). For existing Qubes 4.1-rc1 users, a regular
+[update] is sufficient to upgrade to 4.1-rc2.
+
+In case you haven't heard, Qubes 4.1 includes several major new
+features, each of which is explained in depth in its own article:
+
+- [Qubes Architecture Next Steps: The GUI Domain]
+- [Qubes Architecture Next Steps: The New Qrexec Policy System]
+- [New Gentoo templates and maintenance infrastructure]
+- [Reproducible builds for Debian: a big step forward]
+
+There are also  numerous other improvements and
+bug fixes listed in the [release notes] and in the [issue
+tracker].
+
+Finally, Qubes 4.1 features the following updated default components:
+
+- Xen 4.14
+- Fedora 32 in dom0
+- Fedora 34 template
+- Debian 11 template
+- Whonix 16 Gateway and Workstation templates
+- Linux kernel 5.10
+
+
+How to test Qubes 4.1-rc2
+-------------------------
+
+If you're willing to [test] this release candidate, you can help to
+improve the stable release by [reporting any bugs you encounter].
+Experienced users are strongly encouraged to join the [testing team]!
+
+How to migrate to 4.1-rc2:
+
+- If you're already on 4.1-rc1, simply perform a normal [update].
+- If you're not on 4.1-rc1, you have two options:
+  1. [Back up] your current installation, [download] 4.1-rc2, perform a
+     [fresh install], then [restore] from your backup.
+  2. Perform an [in-place upgrade].
+
+
+Release candidate planning
+--------------------------
+
+As with any initial release candidate, it's likely that user testing
+will reveal important bugs that we'll want to fix before the stable
+release. Depending on the severity of the bugs discovered and how long
+it takes to fix them, we expect that it may be anywhere from a few weeks
+to a few months before we announce the third release candidate. This
```

Comment:
Oh, I haven't spotted it previously - the next rc is at a fixed time: [in 5 weeks](https://www.qubes-os.org/doc/version-scheme/). What isn't set, is how many rc-s will be there. So, it is "few weeks to a few months" to the final release, not the next rc.

Reply:
Thanks. Fixed!